build.sbt

```diff
@@ -19,7 +17,9 @@ lazy val common = Seq(
     </developers>,
   resolvers ++= Seq(
     "Typesafe repository snapshots" at "https://repo.typesafe.com/typesafe/snapshots/",
-    "Typesafe repository releases" at "https://repo.typesafe.com/typesafe/releases/"
+    "Typesafe repository releases" at "https://repo.typesafe.com/typesafe/releases/",
+    // using staging artifact until 2.13 release is ready: https://github.com/apache/logging-log4j-scala/pull/3
+    "Apache staging repository" at "https://repository.apache.org/content/repositories/staging"
```

Comment:
not sure how you want to handle this, but `vimalaguti/logging-log4j-scala` is gone so it can no longer pull in the `log4jScala` project above